Dynatrace/dynatrace-operator
# Cache OneAgent and ActiveGate connection info in Dynakube status

# Description
We want to avoid as many requests to the Dynatrace cluster and to the K8S control plane as possible. Therefore in addtion to the config map, we want to cache the connection info (tenant UUID, token, endpoints, communication hosts) in the Dynakube status. Reconcilers have hold of the DynaKube object anyway, so they don't need to do an extra request to read the required infos from the config map. We can't drop the config map as we need it to map the connection infos to environment variables in the OneAgent daemon set.

## How can this be tested?
Deploy operator and a danykube, check if the connection info config maps are created and if the connection info is part of the dynakube status.

## Checklist
